bpf/bpf_lxc.c

```diff
@@ -1329,6 +1329,7 @@ ipv6_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label,
 	__u32 monitor = 0;
 	__u8 policy_match_type = POLICY_MATCH_NONE;
 	__u8 audited = 0;
+	__s8 ext_err = 0;
```

Comment:
This is passed to `policy_can_access_ingress()` but is not used for anything yet. Maybe this needs to be passed to the caller like in the IPv4 case?

Reply:
thanks, moved it one level up - together with passing it into the `send_drop_notify_ext`